packages/common/src/webrtc/Dialog.ts

```diff
@@ -189,6 +189,11 @@ export default class Dialog {
       case VertoMethod.Attach: {
         const type = NOTIFICATION_TYPE.hasOwnProperty(method) ? NOTIFICATION_TYPE[method] : NOTIFICATION_TYPE.generic
         const notification = { ...params, type, dialog: this }
+
```

Comment:
If you use the `setter` method on Dialog (see comment below) you don't need `node_id` here.